payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionFieldElement.kt

```diff
@@ -7,6 +7,8 @@ import kotlinx.coroutines.flow.Flow
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 sealed interface SectionFieldElement {
     val identifier: IdentifierSpec
+    val shouldRenderOutsideCard: Boolean
```

Comment:
This was needed for `SectionElementUI`, this lets the section know if this element should be rendered outside of the form card. Otherwise all elements get rendered in the card.